public/static/js/spawned_hosts.js

```diff
@@ -1,5 +1,6 @@
 mciModule.controller('SpawnedHostsCtrl', ['$scope', '$window', '$timeout', '$q', 'mciSpawnRestService', 'notificationService', function ($scope, $window, $timeout, $q, mciSpawnRestService, notificationService) {
     $scope.userTz = $window.userTz;
+    $scope.region = $window.region;
```

Comment:
Could you name this with some adjective so that we know whether this is the preferred, selected, or list of available regions?